fairlearn/metrics/_metric_frame.py

```diff
@@ -281,14 +285,15 @@ def __init__(
         self._populate_results(result)
 
         # Handle bootstrapping
-        self._ci_quantiles = None
+        self._ci_quantiles = ci_quantiles
+        self._n_boot = n_boot
```

Comment:
@adrinjalali imho it would safer to keep them private in this case: 

if they are public attributes it might signify that the user is allowed to change them. However if the user changes them after the `__init__` , during which the `result_cache` has been computed, it would  leave the object in an inconsistent state: `self.n_boot`, and `self.ci_quantiles` do not match the computed confidence intervals.

wdyt ?